lib/test_queue/runner/minitest4.rb

```diff
@@ -66,4 +69,20 @@ def run_worker(iterator)
       end
     end
   end
+
+  class TestFramework
+    class MiniTest < TestFramework
+      def all_suite_paths
+        ARGV
+      end
+
+      def suites_from_path(path)
+        ::MiniTest::Unit::TestCase.reset
+        require File.absolute_path(path)
```

Comment:
What happens if we put a directory in `ARGV`? I think this would explode? Would minitest?

Exploding would be fine behavior, but maybe this should be `suites_from_file` if that's the case. Or we could check it for directory-ness explicitly in the frameworks that don't support it.